go.mod

```diff
@@ -40,44 +40,31 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/crypto v0.0.0-20210711020723-a769d52b0f97
-	golang.org/x/net v0.0.0-20210726213435-c6fcb2dbf985
-	golang.org/x/oauth2 v0.0.0-20210628180205-a41e5a781914
+	golang.org/x/net v0.0.0-20210805182204-aaa1db679c0d
+	golang.org/x/oauth2 v0.0.0-20210810183815-faf39c7919d5
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	gomodules.xyz/jsonpatch/v2 v2.2.0
-	google.golang.org/api v0.44.0
+	google.golang.org/api v0.53.0
 	helm.sh/helm/v3 v3.6.3
-	k8s.io/api v0.21.3
-	k8s.io/apiextensions-apiserver v0.21.3
-	k8s.io/apimachinery v0.21.3
-	k8s.io/apiserver v0.21.3
-	k8s.io/cli-runtime v0.21.3
-	k8s.io/client-go v0.21.3
-	k8s.io/code-generator v0.21.3
-	k8s.io/component-base v0.21.3
-	k8s.io/klog/v2 v2.8.0
-	k8s.io/kube-aggregator v0.21.3
+	k8s.io/api v0.22.0
+	k8s.io/apiextensions-apiserver v0.22.0
+	k8s.io/apimachinery v0.22.0
+	k8s.io/apiserver v0.22.0
+	k8s.io/cli-runtime v0.22.0
+	k8s.io/client-go v0.22.0
+	k8s.io/code-generator v0.22.0
+	k8s.io/component-base v0.22.0
+	k8s.io/klog/v2 v2.9.0
+	k8s.io/kube-aggregator v0.22.0
 	k8s.io/kube-openapi v0.0.0-20210527164424-3c818078ee3d
-	k8s.io/kubectl v0.21.3
+	k8s.io/kubectl v0.22.0
 	k8s.io/utils v0.0.0-20210802155522-efc7438f0176
 	rsc.io/letsencrypt v0.0.3 // indirect
-	sigs.k8s.io/controller-runtime v0.9.2
-	sigs.k8s.io/controller-tools v0.6.0
+	sigs.k8s.io/controller-runtime v0.9.6
```

Comment:
I'm interested to know if this speeds up the integration tests. 
Apparently they use a version of etcd which can run with fsync disabled: https://github.com/kubernetes-sigs/controller-runtime/pull/1630